lib/src/render/source_code_renderer.dart

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2019, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:convert';
+
+abstract class SourceCodeRenderer {
```

Comment:
I think the implementations or the abstract class should have a doc comment.

Reply:
Thanks, added some in next commit